.github/workflows/cmake-multi-platform.yml

```diff
@@ -79,6 +79,8 @@ jobs:
         -DFORTE_COM_FBDK=ON
         -DFORTE_COM_HTTP=ON
         -DFORTE_COM_LOCAL=ON
+        -DFORTE_COM_MODBUS=${{ matrix.os == 'windows-latest' && 'OFF' || 'ON' }}
+        -DFORTE_COM_PAHOMQTT=${{ matrix.os == 'windows-latest' && 'OFF' || 'ON' }}
```

Comment:
Wouldn't it be nicer to put the value into a variable and set it in the matrix?

Reply:
Fixed.